draft-ietf-tls-esni.md

```diff
@@ -1717,6 +1717,14 @@ important ideas and contributions.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-tls-esni-11
+
+- Move ClientHello padding to the encoding (#443)
+
+- Relax OuterExtensions checks for alignment with RFC8446 (#467)
+
+- Editorial improvements (#468, #465, #462, #461)
+
```

Comment:
I think we should call out #464 as a bugfix. ECH-11 isn't implementable.

Reply:
Well, not really. It just broke the convention.

Reply:
Fixed anyway.

Reply:
Well, it was a convention break along with an incompatibility with ECH-10. So I guess it's implementable, but you don't want to implement it the way it was written. :-)